tools/tempfs/tempfs.go

```diff
@@ -0,0 +1,36 @@
+package tempfs
```

Comment:
Yeah I think all the usage will probably be underneath tools/ctl

Do you have a suggestion for a different name? localfiles? tempfiles? localcache?

Reply:
I went with localfiles and move it to tools/ctl

Reply:
yeah localfs is less uncomfortable. Then the root dir wouldn't need to be hardcoded here and this could just describe the layout of the fs relative to that configurable root.